submissions/MarharytaBoichenko/html_css_popup/sass/main.scss

```diff
@@ -0,0 +1,304 @@
+$primaryColor: #000;
+$mainFont: 'Merriweather', serif;
+$btnColor: #416aff;
+$bcgColor: #ffffff;
```

Comment:
You can use short #FFF. And it is a good practice to name hex with uppercase

Reply:
thank you, didn`t  know about good practice to name hex with uppercase
but  emmet in  VScode  provides  all  in lowercase( Do  you  know can i change this ?  I cant find about this in settings

Reply:
You are probably right, this is naming convention from my job, don't mind. This is not mandatory :)